src/ctelnet.h

```diff
@@ -244,7 +244,7 @@ public slots:
     int decompressBuffer(char*& in_buffer, int& length, char* out_buffer);
     void reset();
 
-    void processTelnetCommand(const std::string& command);
+    void processTelnetCommand(const std::string& telnetCommand);
```

Comment:
There is no need to provide a name here - it is only really useful when disambiguating multiple arguments of the same type - which isn't needed here...
```suggestion
    void processTelnetCommand(const std::string&);
```

Reply:
Technically no, but the human reading the code would like to know...